src/Rector/MethodCall/AssertIssetToSpecificMethodRector.php

```diff
@@ -115,12 +137,13 @@ private function refactorPropertyFetchNode(Node $node, PropertyFetch $propertyFe
 
         $newArgs = $this->nodeFactory->createArgs([new String_($name), $propertyFetch->var]);
         $node->args = $this->appendArgs($newArgs, $oldArgs);
+        return $node;
     }
 
     /**
      * @param MethodCall|StaticCall $node
      */
-    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): void
+    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): ?Node
```

Comment:
nullable can be removed: 

```suggestion
    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): Node
```